go.mod

```diff
@@ -1,45 +1,43 @@
 
-go 1.21
+go 1.22.0
 
-// after a release version > 1.1.1, remove this line and update the version in the require block with the latest tag
-replace github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230831050635-df1bd5aae9df
+toolchain go1.22.5
 
 replace (
-	github.com/docker/cli => github.com/docker/cli v24.0.7+incompatible
-	github.com/docker/docker => github.com/docker/docker v24.0.7+incompatible
+	// replace those modules from go.opentelemetry.io to elimilate CVEs
+	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc => go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.53.0
+	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.53.0
+	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc => go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.28.0
 
-	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc => go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.46.0
-	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.46.0
-	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc => go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracegrpc v1.23.1
-	golang.org/x/crypto => golang.org/x/crypto v0.17.0
-
-	k8s.io/api => k8s.io/api v0.27.10
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.27.10
-	k8s.io/apimachinery => k8s.io/apimachinery v0.27.10
-	k8s.io/apiserver => k8s.io/apiserver v0.27.10
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.27.10
-	k8s.io/client-go => github.com/rancher/client-go v1.27.4-rancher1
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.27.10
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.27.10
-	k8s.io/code-generator => k8s.io/code-generator v0.27.10
-	k8s.io/component-base => k8s.io/component-base v0.27.10
-	k8s.io/component-helpers => k8s.io/component-helpers v0.27.10
-	k8s.io/controller-manager => k8s.io/controller-manager v0.27.10
-	k8s.io/cri-api => k8s.io/cri-api v0.27.10
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.27.10
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.27.10
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.27.10
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.27.10
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.27.10
-	k8s.io/kubectl => k8s.io/kubectl v0.27.10
-	k8s.io/kubelet => k8s.io/kubelet v0.27.10
-	k8s.io/kubernetes => k8s.io/kubernetes v1.27.10
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.27.10
-	k8s.io/metrics => k8s.io/metrics v0.27.10
-	k8s.io/mount-utils => k8s.io/mount-utils v0.27.10
-	k8s.io/pod-security-admission => k8s.io/pod-security-admission v0.27.10
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.27.10
+	k8s.io/api => k8s.io/api v0.29.7
```

Comment:
Asking because I am unsure, but is bumping to 1.29 going to cause an issues with the previous versions? We have to support 1.27 through 1.30 in system-agent for 2.9 at least.